src/components/Contacts/ContactDetails/ContactDontationsTab/ContactDonationsTab.graphql

```diff
@@ -26,6 +24,14 @@ query GetContactDonations($accountListId: ID!, $contactId: ID!) {
 }
 
 fragment DonationsContact on DonationConnection {
+  pageInfo {
+    endCursor
+    hasNextPage
+    hasPreviousPage
+    startCursor
+  }
+  totalCount
+  totalPageCount
```

Comment:
Any of these unused? Seems like some of them might be after your changes but maybe not.